pkg/controllers/disruption/consolidation.go

```diff
@@ -85,6 +85,11 @@ func (c *consolidation) markConsolidated() {
 
 // ShouldDisrupt is a predicate used to filter candidates
 func (c *consolidation) ShouldDisrupt(_ context.Context, cn *Candidate) bool {
+	// skip any candidates that we can't determine the instance type of
```

Comment:
I guess there's some amount of capacity type filtering but it might be more clear if we gave a bit more info on what we were using this information for in the first place